Beagle-PSE/Beagle
# PalladioFileShare System Test

This PR introduces a Launch Configuration that will automatically execute SoMoX and Beagle on PalladioFileShare, an evaluation project of the SDQ. The test classes are in the root project.

The PR additionally fixes the issue where the command line of “Beagle in Eclipse” could get too long on Windows.

This PR needs 5 lgtm, because I think it’s crucial that everybody can run these tests.

To test it please do at least the following

 * Checkout this PR
 * Run gradle
 * Run the “Beagle in Eclipse” launch configuration
    * You should get an inner Eclipse.
    * You should be able to execute SoMoX in the inner Eclipse.
    * You should be able to run Beagle in the inner Eclipse.
 * Run the “Beagle PalladioFileShare System Test” launch configuration (you’ll propably have to access it through Run > Run Configurations)
    * An inner Eclipse should open
    * SoMoX should be executed in the inner Eclipse
    * Beagle should be executed in the inner Eclipse
    * at the time being, the test does not terminate
    * There will be a lot of red output on the outer console. Ignore it.


Please post whether the above works for you.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Review on Reviewable"/>](https://reviewable.io/reviews/beagle-pse/beagle/804)
